.github/workflows/solidity-tracability.yml

```diff
@@ -0,0 +1,135 @@
+    name: Enforce Traceability
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      id-token: write
+      contents: read
+      pull-requests: write
+    steps:
+      # https://github.com/planetscale/ghcommit-action/blob/c7915d6c18d5ce4eb42b0eff3f10a29fe0766e4c/README.md?plain=1#L41
+      #
+      # Include the pull request ref in the checkout action to prevent merge commit
+      # https://github.com/actions/checkout?tab=readme-ov-file#checkout-pull-request-head-commit-instead-of-merge-commit
+      - name: Checkout the repo
+        uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # v4.1.2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Assume role capable of dispatching action
+        uses: smartcontractkit/.github/actions/setup-github-token@ef78fa97bf3c77de6563db1175422703e9e6674f # setup-github-token@0.2.1
+        id: get-gh-token
+        with:
+          aws-role-arn:  ${{ secrets.AWS_OIDC_CHAINLINK_CI_AUTO_PR_TOKEN_ISSUER_ROLE_ARN }}
+          aws-lambda-url: ${{ secrets.AWS_INFRA_RELENG_TOKEN_ISSUER_LAMBDA_URL }}
+          aws-region: ${{ secrets.AWS_REGION }}
+
+      - name: Make a comment
+        uses: thollander/actions-comment-pull-request@fabd468d3a1a0b97feee5f6b9e499eab0dd903f6 # v2.5.0
+        with:
+          message: |
+            I see you updated files related to `contracts`. Please run `pnpm changeset` in the `contracts` directory to add a changeset.
+          reactions: eyes
+          comment_tag: changeset-contracts
+          # If the changeset is added, then we delete the comment, otherwise we add it.
+          mode: ${{ needs.files-changed.outputs.changesets == 'true' && 'delete' || 'upsert' }}
+          # We only create the comment if the changeset is not added
+          create_if_not_exists: ${{ needs.files-changed.outputs.changesets == 'true' && 'false' || 'true' }}
+
+      - name: Check for new changeset for contracts
+        if: ${{ needs.files-changed.outputs.changesets == 'false' }}
+        shell: bash
+        run: |
+          echo "Please run pnpm changeset to add a changeset for contracts."
+          exit 1
+
+      - name: Setup NodeJS
+        uses: ./.github/actions/setup-nodejs
+
+      - name: Setup Jira
+        working-directory: ./.github/scripts/jira
+        run: pnpm i
+
+        # Because of our earlier checks, we know that both the source and changeset files have changed
+      - name: Enforce Traceability
+        working-directory: ./.github/scripts/jira
+        run: |
+          echo "COMMIT_MESSAGE=$(git log -1 --pretty=format:'%s')" >> $GITHUB_ENV
+          pnpm issue:enforce
+        env:
+          CHANGESET_FILES: ${{ needs.files-changed.outputs.changesets_files }}
+
+          PR_TITLE: ${{ github.event.pull_request.title }}
+          BRANCH_NAME: ${{ github.event.pull_request.head.ref }}
+
+          JIRA_HOST: https://smartcontract-it.atlassian.net/ 
```

Comment:
same here

Reply:
There's no point in having it as a secret, the host is exposed everywhere already. I changed it to be plain text because we post the URL in the `solidity-foundry-artifacts` workflow as part of the step summary